src/libraries/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -133,6 +128,25 @@
     <Compile Include="$(CommonPath)System\Threading\Tasks\TaskToApm.cs"
              Link="Common\System\Threading\Tasks\TaskToApm.cs" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+    <Reference Include="Microsoft.Win32.Primitives" />
```

Comment:
are all of those needed?

Reply:
Yes, I added them by hand. Only necessary ones are in the list.